octue/exceptions.py

```diff
@@ -68,3 +68,7 @@ class BrokenSequenceException(OctueSDKException):
 class InvalidTagException(OctueSDKException, ValueError):
     """ Raise when a tag applied to a data file or dataset
     """
+
+
+class ServiceNotFound(OctueSDKException):
+    pass
```

Comment:
Docstring for what this exception means? There are multiple interpretations of ServiceNotFound, such as:

-  "I received a 404 when attempting to communicate with this service over HTTP(S)"
- "I attempted to communicate with this service using the prescribed protocol but received an error that suggests the service is not present"
- "There's an entry in the `twine` but no consistent service found in the `children.json` file"